core/src/main/java/bio/terra/pearl/core/service/export/formatters/module/SurveyFormatter.java

```diff
@@ -214,7 +214,13 @@ public String getColumnSubHeader(ItemFormatter itemFormatter, boolean isOtherDes
     public static String stripStudyAndSurveyPrefixes(String stableId) {
         // if there are >= 3 underscores, filter out everything before the third underscore
         int thirdUnderscoreIndex = StringUtils.ordinalIndexOf(stableId, "_", 3);
-        if (thirdUnderscoreIndex < 0) {
+
+        // warning: hacky!
+        // the `stableId.toUpperCase()` check is a temporary fix to allow AT's questions
+        // to not be affected by the arcane study stripping logic since many of its questions
+        // feature many underscores. no other study has all caps stableIds, so it's safe(ish)
+        // to do.
+        if (thirdUnderscoreIndex < 0 || stableId.toUpperCase().equals(stableId)) {
```

Comment:
Another slightly less hacky way we could do this is changing the logic to check for the regex `^[a-z]{2}_[a-z]{2}_[a-zA-Z]+_` for detecting prefixes to be stripped.  this would catch that the pattern was always, eg, `oh_oh_medHx_someQuestion`. and it wouldn't strip A-T since the first two strings are lowercase